highfive/newpr.py

```diff
@@ -230,27 +230,46 @@ def is_new_contributor(self, username, owner, repo):
             else:
                 raise e
 
-    def find_reviewer(self, msg):
+    def get_groups(self):
+        groups = deepcopy(self.repo_config['groups'])
+
+        # fill in the default groups, ensuring that overwriting is an
+        # error.
+        global_ = self._load_json_file('_global.json')
+        for name, people in global_['groups'].items():
+            assert name not in groups, "group %s overlaps with _global.json" % name
+            groups[name] = people
+
+        return groups
+
+    def find_reviewer(self, msg, exclude):
         """
         If the user specified a reviewer, return the username, otherwise returns
         None.
         """
         if msg is not None:
             match = reviewer_re.search(msg)
-            return match.group(1) if match else None
+            if match:
```

Comment:
Do we need an `else: return None` to go with this `if`? Sorry, I don't know anything about Python 😄

Reply:
No, if we don't `return` Python considers we're returning `None` implicitely